utils/create_dataset.py

```diff
@@ -104,3 +114,155 @@ def __getitem__(self, index):
                                 f"\nIgnore if some augmentations have padded with dontcare value.")
         sample['index'] = index
         return sample
+
+
+class VRTDataset(GeoDataset):
```

Comment:
Also, please add unit test for the two new classes. You can adapt tests for dataset classes that already exist on torchgeo:
https://github.com/microsoft/torchgeo/tree/main/tests/datasets

Reply:
done, unit tests are implemented in **dataset/test_datasets.py**